tests/bson-serialize-cursor-id.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+MongoDB\Driver\CursorID serialization
+--FILE--
+<?php
+include dirname(__FILE__) . '/utils.inc';
+
+$m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
+cleanup( $m );
+
+$bw = new MongoDB\Driver\BulkWrite( [ 'ordered' => false ] );
+$_id = $bw->insert( [ 'test' => 'foo', 'i' => 1 ] );
+$_id = $bw->insert( [ 'test' => 'foo', 'i' => 2 ] );
+$_id = $bw->insert( [ 'test' => 'foo', 'i' => 3 ] );
+$result = $m->executeBulkWrite( 'demo.test', $bw );
+
+$q = new MongoDB\Driver\Query( [], [ 'batchSize' => 2 ] );
+$cursorId = $m->executeQuery( 'demo.test', $q )->getId();
+var_dump( MongoDB\BSON\toJson( MongoDB\BSON\fromPHP( [ 'cid' => $cursorId ] ) ) );
```

Comment:
I think it might make more sense to bump the hex output of `fromPHP()` (I assume you have the `hex_dump()` helpers found in PHPC). The current expected output will only end up breaking once you upgrade libbson (see: [PHPC-941](https://jira.mongodb.org/browse/PHPC-941)), as the value will end up wrapped in `$numberLong`.